CMakeLists.txt

```diff
@@ -4,7 +4,18 @@ cmake_minimum_required(VERSION 3.4)
 
 include(ExternalProject)
 
-project( CCTag )
+project(CCTag VERSION 1.0.0 LANGUAGES C CXX)
```

Comment:
It's important to have the version in the code, so users of the library can do some `#if CCTAG_VERSION_MAJOR > 1`.
Usually, I create a "version.hpp" and parse the values of major, minor, micro in cmake from this header.